spec/models/repository/git_spec.rb

```diff
@@ -63,7 +63,7 @@
     context 'with managed config' do
       let(:config) { { manages: managed_path } }
       let(:project) { FactoryGirl.build :project }
-      let(:identifier) { "#{project.identifier}.git" }
+      let(:identifier) { project.identifier + ".git" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.